DTDL/rest-stream-resttopic.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:stream:resttopic;1",
+  "@type": "Interface",
+  "displayName": "REST FixedId Stream",
+  "extends": "dtmi:io:tributech:stream:restbase;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "DataStreamID",
```

Comment:
Data Stream Id is not necessary because it will use the dtmi ID which is generated automatically.